### ISSUE TITLE
can not log full cmdline that longer than 4096 chars? 

### ISSUE THREAD

**Issue by QGB** [ColinIanKing/forkstat#4]
long cmdline truncated

**Comment by QGB** [comment]
```
static char *proc_cmdline(const pid_t pid)
{
	int fd;
	ssize_t ret;
	char buffer[65536];

```

**Comment by ColinIanKing** [comment]
Expanded to 64K, using heap instead of stack because it's quite large now. Fix committed 8cfeaec0f0b8643a4457869ae5ff1c3743c950ff
